MacrobenchmarkSample/build.gradle

```diff
@@ -17,23 +17,23 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        agpVersion = "7.1.2"
+        agpVersion = '7.3.0-beta01'
```

Comment:
Are we ok with using beta AGP since it's not compatible with stable AS.
```
The project is using an incompatible version (AGP 7.3.0-beta01) of the Android Gradle plugin. Latest supported version is AGP 7.2.0
```

There was some discussion in the past, that we should use stable AGP in samples to prevent issues with opening the samples.